extensions/roc-package-web-app-react-dev/package.json

```diff
@@ -35,6 +35,7 @@
     "redux-logger": "~2.6.1",
     "roc": "^1.0.0-rc.12",
     "roc-package-web-app-dev": "^1.0.0-beta.3",
+    "roc-package-web-app-react": "file:../roc-package-web-app-react",
```

Comment:
This is an error when using `rid` with npm 5, could you remove this? 🙂 

_This will not be a problem when we switch over to the new and improved tooling in `roc-plugin-repo`._